### PR TITLE
[DEV APPROVED] Remove signup flow

### PR DESCRIPTION
**Pull request by neoeno**
Requires https://github.com/moneyadviceservice/mas-rad_core/pull/83

The last PR (#299) added onboarding to the self-service. We needed that because we're removing the sign-up flow. This PR removes it all, and also does a few other bits like reorganising the locales and removing newly-redundant styles (because they were used only in the sign-up section).

Have at it!

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -17,6 +17,8 @@ module Rad
   class Application < Rails::Application
     config.autoload_paths << Rails.root.join('lib')
     config.active_record.raise_in_transactional_callbacks = true
+    config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}')]
```
Comment (mconnell) [inline config/application.rb]: I'm guessing the partials in the /self_service/ directory weren't autoloading?

**Comment by benbarnett** [comment]
:+1:

File: config/application.rb (hunk 1 above)
Comment (neoeno) [inline config/application.rb]: you guess correct

File: db/schema.rb [hunk 2]
```diff
@@ -31,14 +31,13 @@
...
-    t.boolean  "confirmed_disclaimer",              null: false
-    t.string   "postcode",             default: "", null: false
-    t.integer  "travel_distance",      default: 0,  null: false
+    t.string   "reference_number",              null: false
+    t.string   "name",                          null: false
+    t.integer  "firm_id",                       null: false
+    t.datetime "created_at",                    null: false
+    t.datetime "updated_at",                    null: false
+    t.string   "postcode",         default: "", null: false
+    t.integer  "travel_distance",  default: 0,  null: false
```
Comment (neoeno) [inline db/schema.rb]: Schema's updated here because https://github.com/moneyadviceservice/mas-rad_core/pull/83 removes `confirmed_disclaimer`

**Comment by mconnell** [comment]
:+1:

**Comment by munckymagik** [comment]
:+1:

**Comment by neoeno** [comment]
Thanks everyone!

**Pull request merged**